gpu_ocean/SWESimulators/Common.py

```diff
@@ -714,9 +789,12 @@ def __init__(self, gpu_ctx, nx, ny, \
         self.linearInterpolation_EW = self.boundaryKernels.get_function("linearInterpolation_EW")
         self.linearInterpolation_EW.prepare("iiiiiiiiPiPiPi")
         self.flowRelaxationScheme_NS = self.boundaryKernels.get_function("flowRelaxationScheme_NS")
-        self.flowRelaxationScheme_NS.prepare("iiiiiiiiPiPiPi")
+        self.flowRelaxationScheme_NS.prepare("iiiiiiiiPiPiPif")
```

Comment:
added "time" interpolation as new kernel argument to linearly interpolate in time the boundary conditions